src/gui/tray/ActivityItemContent.qml

```diff
@@ -95,13 +95,13 @@ RowLayout {
         Label {
             id: activityTextTitle
             text: (root.activityData.type === "Activity" || root.activityData.type === "Notification") ? root.activityData.subject : root.activityData.message
+            height: (text === "") ? 0 : implicitHeight
```

Comment:
why not making it not visible if `text` is empty ?

Reply:
Sure, added the visible prop